conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/bigmaps/BigMapsOperations.scala

```diff
@@ -388,7 +388,8 @@ case class BigMapsOperations[Profile <: ExPostgresProfile](profile: Profile) ext
                 balance = BigDecimal(tokenUpdate.balance),
                 blockId = blockData.hash.value,
                 blockLevel = blockData.header.level,
-                asof = toSql(blockData.header.timestamp)
+                asof = toSql(blockData.header.timestamp),
+                forkId = Fork.mainForkId
```

Comment:
I'm probably missing something, but how do we verify that this will be `mainForkId`?

Reply:
never mind 😄 I just realised it will be invalidated when the time comes.